justfile

```diff
@@ -73,8 +73,8 @@ env:
     cp ingestion_server/env.template ingestion_server/.env
 
 # Load sample data into the Docker Compose services
-init: up wait-for-es wait-for-ing wait-for-web
-    ./load_sample_data.sh
+init log_level="INFO": up wait-for-es wait-for-ing wait-for-web
+    env LOG_LEVEL={{ log_level }} pipenv run python load_sample_data.py
```

Comment:
There needs to be an install step before `load_sample_data`, because currently it breaks because `decouple` is not installed:
```
    from decouple import config
ModuleNotFoundError: No module named 'decouple'
```